AppCenterDataStorage/AppCenterDataStorage/Internal/MSDataStorePrivate.h

```diff
@@ -16,7 +16,7 @@ static NSString *const kMSDefaultApiUrl = @"https://api.appcenter.ms/v0.1";
 /**
  * A flag that indicates offline mode is on or off.
  */
-@property(atomic) BOOL isOfflineModeEnabled;
+@property(atomic) BOOL offlineModeEnabled;
```

Comment:
Just to strictly respect the convention.
```suggestion
@property(atomic, getter=isOfflineModeEnabled) BOOL offlineModeEnabled;
```